packages/server/src/server.ts

```diff
@@ -64,14 +69,29 @@ registerHistoryApi(app, DbConnection.getInstance(), pgp)
 
 const httpServer = http.createServer(app)
 
-const serverPort = parseInt(process.env.NODE_PORT || "3005")
+const serverPort = ServerConfig.getInstance().serverPort()
+
+// Initialize database
```

Comment:
I wouldn't expect these kinds of code changes for a PR that's about (changing) structured logging. Should this be here?

Reply:
The logging needed many more configuration options, therefore I also adjusted the way configuration should be done. So yes, it should be here :-), and yes, it is two changes in the same PR :-(.

Reply:
I meant more the DB initialization code from line 74 onwards, btw.